module/tools/SystemConfiguration/data/config/SystemConfiguration.yaml

```diff
@@ -35,7 +35,7 @@ systemd:
   - systemd-resolved.service
   - systemd-timesyncd.service
   - sshd.service
-  - wpa_supplicant@wlp58s0.service
+  - wpa_supplicant@wlp0s20f3.service
```

Comment:
Do we know why this is `wlp0s20f3` and not  `wlan0`? I have no understanding on any of these interface names, just curious as we changed from `WIFI_INTERFACE:-"wlp58s0"` to `WIFI_INTERFACE=${WIFI_INTERFACE:-"wlan0"}`

Reply:
The next line in `arch-chroot_install.sh` runs `udevadm test-builtin net_id` to find the [predictable network name](https://systemd.io/PREDICTABLE_INTERFACE_NAMES/) udev will give the wlan0 device. For the nuc12s, this is `wlp0s20f3`. After running the scripts and rebooting, the `99-default.link` file should tell udev to use the predictable network names (now the standard) so the device which was `wlan0` in the live install environment is now `wlp0s20f3`.

Reply:
Since you can give interface names as environment variables to  `arch-chroot_install.sh` now, using whatever the names are in the installation environment should be fine. `udevadm` will find the right name.